draft-ietf-httpbis-digest-headers.md

```diff
@@ -398,6 +372,32 @@ Two examples of its use are
            id-sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=
 ~~~
 
+## Computing Digests of Representation Data {#representation-digest}
+
+A representation digest is the value of the output of a digest algorithm,
+together with an indication of the algorithm used (and any parameters).
+
+~~~
+   representation-data-digest = digest-algorithm "="
+                                <encoded digest output>
+~~~
```

Comment:
I think there is an opportunity to merge this art with the art for the definition of Digest Header. That way it becomes clear exactly what composes the Digest header, rather than needing to flick back and forth.

Reply:
Yes, that's in another PR :D